server/utils.py

```diff
@@ -0,0 +1,15 @@
+# utils.py
+import uuid
+import os
+import glob
+
+def random_tts_filename ():
+    filetext = str(uuid.uuid4())
+    filename = "files/" + filetext + ".mp3"
+    return filename
```

Comment:
Flask expects any files accessible from the front-end to be within the path we set in `app(static_folder='../some_path')`

So let's prefix `files` with the static folder path. Usually that will be a global variable defined in a `constant.py` file and then imported and used wherever it's needed, in this case files `app.py`, `utils.py`.

But since the project is small,  feel free to hard-code it. just add the static folder path to the string:
` filename = "../static/build/statics/files/" + filetext + ".mp3"`